pkg/bkctl/bookie/list_disk_file.go

```diff
@@ -0,0 +1,85 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package bookie
+
+import (
+	"github.com/streamnative/pulsarctl/pkg/bookkeeper/bkdata"
+	"github.com/streamnative/pulsarctl/pkg/cmdutils"
+)
+
+func listDiskFileCmd(vc *cmdutils.VerbCmd) {
```

Comment:
since it is listing the files, does it support pagination?

Reply:
This is the result of the list files:
```
{
    "journal files" : "filename1 filename2 ...",
    "entrylog files" : "filename1 filename2...",
    "index files" : "filename1 filename2 ..."
  }
```

Should we need to support pagination?

Reply:
I am asking if the restful API support pagination. If the API supports pagination, we should support pagination in the tool as well, no?

Reply:
Oh. Sorry for my misunderstand. The service does not support the pagination.